content/en/blog/_posts/2024-04-xx-structured-authz-beta.md

```diff
@@ -0,0 +1,328 @@
+---
+layout: blog
+title: 'Multi-Webhook and Modular Authorization Made Easy'
+date: 2024-04-nn
```

Comment:
```suggestion
date: 2024-04-25
```
Please add the assigned publication date

Reply:
(this also means a file rename, I'm afraid)